### ISSUE TITLE
TypeError: 'NoneType' object is not iterable (since cyclonedx-bom==3.6.4)

### ISSUE THREAD

**Issue by eudyptula** [CycloneDX/cyclonedx-python#440]
Running `cyclonedx-bom --format json -e -o bom.json` since version 3.6.4 gives us the following exception - doesn't happen with 3.6.3 and earlier:

```
Traceback (most recent call last):
  File "/usr/local/bin/cyclonedx-bom", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/site-packages/cyclonedx_py/client.py", line 307, in main
    CycloneDxCmd(args).execute()
  File "/usr/local/lib/python3.10/site-packages/cyclonedx_py/client.py", line 147, in execute
    output = self.get_output()
  File "/usr/local/lib/python3.10/site-packages/cyclonedx_py/client.py", line 84, in get_output
    parser = self._get_input_parser()
  File "/usr/local/lib/python3.10/site-packages/cyclonedx_py/client.py", line 254, in _get_input_parser
    return EnvironmentParser()
  File "/usr/local/lib/python3.10/site-packages/cyclonedx_py/parser/environment.py", line 78, in __init__
    for classifier in i_metadata.get_all("Classifier"):
TypeError: 'NoneType' object is not iterable
```

**Comment by jkowalleck** [comment]
Thank you so much for the bug report,

@eudyptula 

a fix will be available soon

**Comment by jkowalleck** [comment]
the issue should be fixed via v3.7.1
Could you test and confirm, @eudyptula  ?

**Comment by eudyptula** [comment]
@jkowalleck, no exception anyone.

Thanks for the super quick fix :)
